### PR TITLE
Add auto_conf to agent for cluster agent stats.

### PR DESCRIPTION
**Pull request by devonboyer**
### What does this PR do?

Add auto_conf to agent for cluster agent stats.

### CODE REVIEW

File: cmd/cluster-agent/api/v1/install.go [hunk 1]
```diff
@@ -10,28 +10,25 @@ import (
...
+	metadataStats    = new(expvar.Map).Init()
+	metadataErrors   = &expvar.Int{}
+	metadataRequests = &expvar.Int{}
 )
 
 func init() {
 	apiStats.Set("Metadata", metadataStats)
 	metadataStats.Set("Errors", metadataErrors)
-	metadataStats.Set("RequestsPerSecond", metadataRequestsPerSecond)
+	metadataStats.Set("Requests", metadataRequests)
```
Comment (devonboyer) [inline cmd/cluster-agent/api/v1/install.go]: These need to be monotonic counters and Datadog will compute the rate.

**Review by CharlyF — DISMISSED** [review]
🌴

**Comment by devonboyer** [comment]
This appears to have caused issues with agents connecting to the DCA on port 5005.

**Comment by CharlyF** [comment]
Issue with 5005 was caused is fixed here https://github.com/DataDog/datadog-agent/pull/2205. You can go for it.

**Pull request closed without merging**